backend/handlers/handlers.go

```diff
@@ -971,7 +971,8 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 				return result[i:]
```

Comment:
I'm a bit allergic to comparing float values with 0.
not directly related to this commit but anyway, `if e.Value > 0 { return result[i:] }` is better imho.

Reply:
done